editors/code/package.json

```diff
@@ -39,21 +39,21 @@
         "vscode-languageclient": "7.0.0-next.1"
     },
     "devDependencies": {
-        "@rollup/plugin-commonjs": "^11.1.0",
-        "@rollup/plugin-node-resolve": "^7.1.3",
+        "@rollup/plugin-commonjs": "^12.0.0",
+        "@rollup/plugin-node-resolve": "^8.0.0",
         "@types/glob": "^7.1.1",
         "@types/mocha": "^7.0.2",
-        "@types/node": "^12.12.39",
+        "@types/node": "^14.0.5",
```

Comment:
I've just noticed, that this is not what we want to do. Vscode is using nodejs12, @lnicola I recommend leaving it under v12 major to keep us safe